frontend/src/components/utils/ProgressSpinner.module.scss

```diff
@@ -0,0 +1,11 @@
+.container {
+  margin-top: 60px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 60px;
+}
+
+.spinner {
+  height: 60px !important;
```

Comment:
Try to avoid `!important` when styling things. It usually means that you can change this CSS property somewhere else (maybe try the container div?)